src/main/antlr/SHRContentProfileParser.g4

```diff
@@ -8,10 +8,10 @@ doc:                docHeader contentsDefs;
 docHeader:          KW_GRAMMAR KW_G_CONTENT_PROFILE version;
 
 contentsDefs:       (namespaceHeader contentDefs)*;
-namespaceHeader:    KW_NAMESPACE namespace;
+namespaceHeader:    KW_NAMESPACE namespace (namespaceFlag)?;
```

Comment:
The parens `(` `)` are not harmful, but seem unnecessary...  I'm guessing that maybe you originally had an `|` in there but it got removed?

Reply:
Fixed in https://github.com/standardhealth/shr-grammar/pull/45/commits/56b015afeaf7aa9d489026ce54c8aa4011a79ecf.